src/airport.js

```diff
@@ -1 +1,78 @@
+class Airport {
+    listOfPlanes;
+    airportCapacity;
+    currentWeather;
+    notPlaneError = new Error("Object is not a plane");
+    atCapacityError = new Error("The airport is at capacity");
+    notAtAirportError = new Error("That plane is not at the airport");
+    capacityFormatError = new Error("Invalid capacity");
+    alreadyLandedPlaneError = new Error("Cannot land an already landed plane");
+    tooStormyError = new Error("The weather is too bad to fly");
```

Comment:
I liked having all of the possible errors in one place, named thisIsTheCauseError. Makes it clear why the error was called